src/plopp/functions/superplot.py

```diff
@@ -66,10 +66,53 @@ def remove_line(self, change: Dict[str, Any], line_id: str):
         self._update_container()
 
 
+class Superplot:
+    """
+    Object that slicing all but one dimension of the input, with a slider per sliced
```

Comment:
```suggestion
    Object that slices all but one dimension of the input, with a slider per sliced
```

Reads a bit strange. Why 'Object'? Is this a figure or a component of a figure or how is this supposed to be used?

Reply:
Mainly, it's just an implementation detail which makes it easier to write unit tests. When running the unit tests, we are not in a jupyter notebook, and figures are thus not widgets. We cannot embed them in the `Box` at the end of the `superplot` function. So it's easier to put most of the code in this class, which can be tested,

The `superplot` function is then just a thin wrapper which is not covered by the unit tests.

It's not meant to be used anywhere outside of the `superplot` function, which is why I struggled to find a good name/description.

Reply:
I added comments in the docstrings about this.